### PR TITLE
chore: replace TravisCI with spec-prod

### PR DESCRIPTION
**Pull request by marcoscaceres**
@michael-n-cooper, I believe this should do the same thing as the .travis script. 

Question: any reason not to push to /TR/?

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/accname/pull/133.html" title="Last updated on Jun 25, 2021, 12:42 AM UTC (1f91f1d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/accname/133/5823cd8...1f91f1d.html" title="Last updated on Jun 25, 2021, 12:42 AM UTC (1f91f1d)">Diff</a>

### CODE REVIEW

**Comment by michael-n-cooper** [comment]
There are lots of reasons not to push to TR automatically. I'm on holiday so won't take time to elaborate them now. Please don't push any workflow changes to my repositories. I will look at the Travis -> GH actions change when I'm back, but don't plan in the short term to go further than pushing snapshots to gh-pages and would need lots of process coordination to go further.

**Comment by marcoscaceres** [comment]
@michael-n-cooper, no problem - happy for you to handle it. When you are back, please see the email from @plehegar about our service with Travis ending. If you need a hand, just give me a shout! happy to help. 

Enjoy your break!!! 🏝

File: .github/workflows/auto-publish.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Node CI
+
+on:
+  push:
+    branches:
+      - gh-pages
```
Comment (plehegar) [inline .github/workflows/auto-publish.yml:6]:
```suggestion
      - main
```

File: .github/workflows/auto-publish.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  pull_request: {}
+
+jobs:
+  validate-and-publish:
+    name: Validate and Publish
```
Comment (plehegar) [inline .github/workflows/auto-publish.yml:11]:
```suggestion
    name: Validate and publish to gh-pages
```

**Review by plehegar — CHANGES_REQUESTED** [review]

**Comment by plehegar** [comment]
Note: I didn't review the changes proposed to the JS code regarding respec. Apparently, the current code is broken and needs fixing. @marcoscaceres kindly introduced the fix in this pull request. This pull request could be split in two if it poses issues (ie one to switch to GitHub action and one to switch to fix the respec JS code).

**Review by michael-n-cooper — APPROVED** [review]

**Pull request merged**